src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -428,6 +427,20 @@ private void Visit(StackFrame frame, string framePointer)
         private void Visit(ThreadFlow threadFlow, string threadFlowPointer)
         {
             Analyze(threadFlow, threadFlowPointer);
+
+            if (threadFlow.Locations != null)
+            {
+                ThreadFlowLocation[] threadFlowLocations = threadFlow.Locations.ToArray();
+                string threadFlowLocationsPointer = threadFlowPointer.AtProperty(SarifPropertyName.Locations);
```

Comment:
>threadFlowLocationsPointer [](start = 23, length = 26)

What the heck is this? a string-based pointer to something? [i realize this question is orthogonal to your change :)]

Reply:
The extension method `AtProperty` appends a property name, for instance `/runs[0]` -> `/runs[0]/locations`, and the method `AtIndex` appends an array index, for instance,  `/runs[0]/locations` -> `/runs[0]/locations[1]`

---
In reply to: [205216379](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205216379) [](ancestors = 205216379,205214627)